core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/PresentedOfferingContext.ios.kt

```diff
@@ -1,24 +1,21 @@
 package com.revenuecat.purchases.kmp
 
-import cocoapods.PurchasesHybridCommon.RCPresentedOfferingContext
-import cocoapods.PurchasesHybridCommon.RCTargetingContext
-
-public actual typealias PresentedOfferingContext = RCPresentedOfferingContext
-
-public actual typealias PresentedOfferingTargetingContext = RCTargetingContext
-
-
-public actual val PresentedOfferingContext.offeringIdentifier: String
-    get() = offeringIdentifier()
-
-public actual val PresentedOfferingContext.placementIdentifier: String?
-    get() = placementIdentifier()
-
-public actual val PresentedOfferingContext.targetingContext: PresentedOfferingTargetingContext?
-    get() = targetingContext()
-
-public actual val PresentedOfferingTargetingContext.revision: Int
-    get() = revision().toInt()
-
-public actual val PresentedOfferingTargetingContext.ruleId: String
-    get() = ruleId()
+import cocoapods.PurchasesHybridCommon.RCPresentedOfferingContext as IosPresentedOfferingContext
+import cocoapods.PurchasesHybridCommon.RCTargetingContext as IosPresentedOfferingTargetingContext
+
+public actual class PresentedOfferingContext(
+    internal val wrapped: IosPresentedOfferingContext
+) {
```

Comment:
Ideally this is not an `expect` / `actual` class (see Toni's comment), but if it has to be, the constructor should be `internal`.

Reply:
This also applies to `PresentedOfferingTargetingContext` ,as well as to both classes on Android.